### ISSUE TITLE
NuGet package

### ISSUE THREAD

**Issue by rogerfar** [agrublev/angularLocalStorage#60]
Hi,

I was looking in Nuget but I can't find this package, under what name is it posted?

Thanks!

**Comment by hppycoder** [comment]
I don't believe that it's officially supported on NuGet. One of the authors @agrublev or @CamaroSS would have to create an account - https://www.nuget.org/users/account/LogOn?ReturnUrl=%2Fpackages%2Fupload then add the repo.

**Comment by hppycoder** [comment]
Steps to get a new package setup - http://docs.nuget.org/docs/creating-packages/creating-and-publishing-a-package

Not sure how often we can build, and keep NuGet updated as well. My concern about using this build application is that it would be prone to being behind since ever build has to go through NuGet's build system. This is why Bower is so powerful, it's never out of date and there's nothing that you have to "deploy" to the server.

@rogier21 could you use Bower?

**Comment by rogier21** [comment]
@hppycoder at the moment our project just uses NuGet packages, and I get angularLocalStorage in by hand. I didn't want to start using Bower till the next Visual Studio version, and I would have to see what the impacts are with the build servers.

**Comment by hppycoder** [comment]
@rogier21 - That makes a lot of sense. I will look into getting this setup, it seems rather complicated for something that should be really simple. Bower would be the best setup plan as it's always going to be updated. I worry that if we release the maintainer that releases would have to know to also go through the NuGet setup process and update it there too.
